pydantic/v1/color.py

```diff
@@ -338,7 +338,7 @@ def parse_hsl(h: str, h_units: str, sat: str, light: str, alpha: Optional[float]
 
 
 def float_to_255(c: float) -> int:
-    return int(round(c * 255))
+    return round(c * 255)
```

Comment:
Please remove the `V1` changes.

You can create a separate PR against [1.10.X-fixes](https://github.com/pydantic/pydantic/tree/1.10.X-fixes) and include this change there